sound/soc/sof/intel/pci-lnl.c

```diff
@@ -47,6 +47,7 @@ static const struct sof_dev_desc lnl_desc = {
 	.nocodec_tplg_filename = "sof-lnl-nocodec.tplg",
 	.ops = &sof_lnl_ops,
 	.ops_init = sof_lnl_ops_init,
+	.ipc_timeout = 5000,
```

Comment:
the default is 500ms, that seems plenty for an IPC, no? Moving to a 5s timeout to deal with a suspend-resume problem means suspend-resume is already broken in the first place, regardless of the IPCs...